SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -233,6 +236,106 @@ public bool TryDownloadEmbeddedFile(string pluginKey, string embeddedFileName, s
 
         #region Private methods
 
+        private void doGetServerVersion()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method 'doGetServerVersion' to match camel case naming rules, consider using 'DoGetServerVersion'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS100)